primitives/storage-keys/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "substratee-storage-keys"
+version = "0.1.0"
+edition = "2018"
+resolver = "2"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false}
+
+frame-metadata = { version = "13.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master"}
+sp-core = { version = "3.0.0", default-features = false, features = ["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "master"}
+sp-std = { version = "3.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master"}
```

Comment:
I think I would prefer this kind of dependency format. What do you think @haerdib, @mullefel?

I think it is easier to see what kind of dependencies we have. The other format on the other hand allows distinguishing better the activated feature and stuff, but makes the overall file harder to read.

Reply:
I don't really have a strong preference for either option, to be honest. The more important aspect for me is to have consistency.. so whichever option we choose, we should stick to it and apply it in all tomls

Reply:
I agree, it gives the better general overview and it's less verbouse. So I'm approving.

Reply:
Ok, I made an issue with our conclusion: https://github.com/integritee-network/worker/issues/329